src/core.c/CompUnit/Repository/Installation.pm6

```diff
@@ -532,6 +533,48 @@ sub MAIN(:$name, :$auth, :$ver, *@, *%) {
         }
     }
 
+    # if we have to include :$name then we take the slow path
+    multi method candidates(Str:D :$file!, Str:D :$name!, :$auth, :$ver, :$api) {
```

Comment:
Something else that is weird about including `:$file` and `:$name` in `method candidates` is that elsewhere (`method files`) `$name` refers to the distribution name, but here it refers to a module name.